client/src/App.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
 import './App.css';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Home from './Home.js'
+import Nav from './Nav.js';
+import About from './About.js';
+import Contact from './Contact';
+import Code from './Code.js';
+import Music from './Music.js';
 
 function App() {
+ 
   return (
-    <div className="App">
-      <h1>Hello portfolio!</h1>
-    </div>
+    <Router>
+      <body className="main-wrapper">
+        <Nav></Nav>
+        
+        <img src='./pics/Profile-pic.jpg' id='profile-pic' alt='my stupid mug'></img>
+        <Switch>
+          <Route exact path='/'><Home/> </Route>
+          <Route path='/About'><About /> </Route>
+          <Route path='/Code' component={Code} />
+          <Route path='/Music' component={Music} />
+          <Route path='/Contact' component={Contact} />
+        </Switch>
+      </body>
+    </Router>
```

Comment:
Great use of browser router, and nice work keeping your Navbar outside of your switch since it will be displayed on every page no matter what.

You could make your `<Nav />` self closing though. 

Also using the `component={...}` is better practice then putting your `<Home />` tag in the `<Route></Route>` tags